aiohttp/web_request.py

```diff
@@ -151,15 +151,30 @@ def secure(self):
         return self.url.scheme == 'https'
 
     @reify
+    def _forwarded(self):
+        forwarded = self._message.headers.get(hdrs.FORWARDED)
+        if forwarded is not None:
+            parsed = re.findall(
+                r'^by=([^;]*); +for=([^;]*); +host=([^;]*); +proto=(https?)$',
```

Comment:
Please note: the lase example contains two `for` pairs, it's not supported by your regexp now.